Source/Core/DolphinWX/Frame.cpp

```diff
@@ -1324,9 +1325,17 @@ void CFrame::ParseHotkeys()
 
 	if (!Core::IsRunningAndStarted())
 	{
+		onceOnLaunch = true;
 		return;
 	}
 
+	if (onceOnLaunch)
+	{
+		onceOnLaunch = false;
+		if (SConfig::GetInstance().bAutoLoadSavestate)
+			State::LoadLastSaved();
```

Comment:
PR #4565 fixed an bug on loading a savestate very early. Was this the error you've seen on loading a savestate? At least a hacky State::Load() in the CPUThread worked for me.